File: dotnet/DotNetProblems/SmallProblems/Pyramid.cs
//using System;
//using System.Collections.Generic;
//using System.Linq;
//using System.Text;
//using System.Threading.Tasks;

namespace DotNetProblems.SmallProblems
{
    class PrintPyramidClass{
        public static void PrintPyramid(int lines){
            int spaces = lines + 3;
            for(int i =0;i < lines;i++){
                for(int j=0; j< spaces; j++){
                    Console.Write(" ");
                }

                for(int k=0; k<=i; k++){
                    Console.Write("P ");
                }
                Console.WriteLine("");
                spaces-=1;
            }


        }

        public static void Main(){

            PrintPyramid(5);
        }
    }

}

File: js/pyramid.js
function printPyramid(rows){
    let spaces = rows +3;
    for(var i=0; i<rows;i++){
        for(var j=0;j<spaces - i;j++){
            process.stdout.write(' ');
        }
        for(var k =0; k<i;k++){
            process.stdout.write('P ');
        }
        process.stdout.write('\n');
    }
}

printPyramid(5);